src/utils/dataset_utils.py

```diff
@@ -341,126 +403,6 @@ def get_candidate_stable_id(can):
 ######################################################################################################
 
 # MOST OF THESE CLASSES ARE DIFFS OFF OF EXISTING SNORKEL PREPROCESSORS
```

Comment:
Need `CandidateExtractorUDF` and `CandidateExtractorFilter` classes (see diff)

Reply:
Added.